### ISSUE TITLE
Ios 9 material-icons

### ISSUE THREAD

**Issue by paulux84** [GwtMaterialDesign/gwt-material#327]
On IOS 9 material-icons are displayed wrong.

![image](https://cloud.githubusercontent.com/assets/7997728/15476896/2efc6b08-2112-11e6-9ff0-5819eca595d9.png)

As you can see an unexpected space is rendered between icon and text.

Another example...

![image](https://cloud.githubusercontent.com/assets/7997728/15476929/5777035e-2112-11e6-82b5-bd193abfd8ad.png)

...and so on...

This bug is obviously reflected on all the components that have an icon.

**Comment by BenDol** [comment]
What version of GMD are you using?

**Comment by paulux84** [comment]
1.5.0 release
But my screenshot are on http://gwtmaterialdesign.github.io/gwt-material-demo/snapshot/

**Comment by MarcoCucchi** [comment]
I have the same problems with OSX El capitan/Safari 9.0.3. I took some screenshots
(http://gwtmaterialdesign.github.io/gwt-material-demo/snapshot)

![schermata 2016-05-23 alle 18 22 57](https://cloud.githubusercontent.com/assets/13817916/15477516/5aa930da-2115-11e6-8a55-4e31e41b7f52.png)
![schermata 2016-05-23 alle 18 25 39](https://cloud.githubusercontent.com/assets/13817916/15477517/5ac5c218-2115-11e6-9e7b-d8081171c498.png)
![schermata 2016-05-23 alle 18 25 49](https://cloud.githubusercontent.com/assets/13817916/15477522/5ada8112-2115-11e6-9f8d-019f796de787.png)
![schermata 2016-05-23 alle 18 26 09](https://cloud.githubusercontent.com/assets/13817916/15477518/5acb244c-2115-11e6-90a6-ad583a11f177.png)
![schermata 2016-05-23 alle 18 26 21](https://cloud.githubusercontent.com/assets/13817916/15477521/5ad7e2d6-2115-11e6-9904-405b80030ac0.png)
![schermata 2016-05-23 alle 18 26 40](https://cloud.githubusercontent.com/assets/13817916/15477519/5ad15cea-2115-11e6-9983-2aa0848234bc.png)
![schermata 2016-05-23 alle 18 27 46](https://cloud.githubusercontent.com/assets/13817916/15477520/5ad2703a-2115-11e6-8fe1-eb9546a58315.png)
![schermata 2016-05-23 alle 18 28 18](https://cloud.githubusercontent.com/assets/13817916/15477523/5ade2a74-2115-11e6-91b0-b572cadbcfbe.png)
![schermata 2016-05-23 alle 18 29 51](https://cloud.githubusercontent.com/assets/13817916/15477524/5ae69ea2-2115-11e6-93f8-33d23b9e846f.png)
![schermata 2016-05-23 alle 18 30 23](https://cloud.githubusercontent.com/assets/13817916/15477525/5aef692e-2115-11e6-8e38-8d92a18f5a1c.png)
![schermata 2016-05-23 alle 18 30 58](https://cloud.githubusercontent.com/assets/13817916/15477527/5af7a8aa-2115-11e6-95f9-cfe1f7e99426.png)
![schermata 2016-05-23 alle 18 31 24](https://cloud.githubusercontent.com/assets/13817916/15477526/5af463de-2115-11e6-95ac-a16db4b181ae.png)
![schermata 2016-05-23 alle 18 32 14](https://cloud.githubusercontent.com/assets/13817916/15477528/5afac60c-2115-11e6-83bc-21bbb9ec2084.png)

**Comment by BenDol** [comment]
Was this working previously?

**Comment by paulux84** [comment]
I don't know...i've never tried IOS 9 before.
a really strange things is that sometimes on load all is displayed correctly...but if i do something (for example scroll) the bug appears

**Comment by MarcoCucchi** [comment]
My fault, this wasn't working previously, I did my test on Osx/Chrome not with Safari. (on previous similar bug #309)

**Comment by BenDol** [comment]
I have tested this on Safari 9.1.1 and it works perfectly fine there. If updating Safari isn't a good solution a PR with a fix would be greatly appreciated! That being said I tested it on Safari 7.x too without any problems. Seems like an issue in a Safari build.

**Comment by paulux84** [comment]
@BenDol the problem is on IO9...only for mobile for me (example ipad/iphone)
i've tested with various browser and ios version (i've premium version of browserstack) and the bug exist in  1.5.0 http://gwtmaterialdesign.github.io/gwt-material-demo/ and 1.5.1 http://gwtmaterialdesign.github.io/gwt-material-demo/snapshot/.
I don't know if the same problem is present in previous version, but if i do the same test in materializecss.com all works great.
I've tried many solution with browserstack developer tools, but nothings seems to have any effect.

**Comment by paulux84** [comment]
@BenDol in some devices safari works great but the problem appears in chrome -.-'

**Comment by paulux84** [comment]
@BenDol  EUREKA!
On IOS 9 the browser set the width of the <i> as if to render the inner text.
So for example the material icon "polymer " width is bigger then material icon "book"...panic!!!

**Comment by paulux84** [comment]
And why in materializecss.com this bug is not present??

**Comment by BenDol** [comment]
Strange indeed, is there an advised solution out there?

**Comment by paulux84** [comment]
no...i can only say that if i remove "font-family: Material Icons" the width is correct... arrrrr

**Comment by BenDol** [comment]
Can you test the [snapshot demo](http://gwtmaterialdesign.github.io/gwt-material-demo/snapshot) now with my latest changes?

**Comment by paulux84** [comment]
if your changes are on this url http://gwtmaterialdesign.github.io/gwt-material-demo/snapshot...still not working!
if i need to compile it from master and try locally... you must wait tomorrow morning...i must go...

**Comment by paulux84** [comment]
I must compile locally? 

Il 23 maggio 2016 20:54:57 CEST, Ben Dol <notifications@github.com> ha scritto:
>Can you test the [snapshot
>demo](http://gwtmaterialdesign.github.io/gwt-material-demo/snapshot)
>now with my latest changes?
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/GwtMaterialDesign/gwt-material/issues/327#issuecomment-221056946

-- 
Inviato dal mio dispositivo Android con K-9 Mail. Perdonate la brevità.

**Comment by paulux84** [comment]
Goodmorning,
i've done another little step towards the solution...
if i remove ```<link href="https://fonts.googleapis.com/icon?family=Material+Icons"
      rel="stylesheet">``` and directly import ```<link href="https://fonts.googleapis.com/icon?family=Material+Icons"
      rel="stylesheet">``` it works...

**Comment by paulux84** [comment]
the solution is something here 
```
/* fallback */
@font-face {
  font-family: 'Material Icons';
  font-style: normal;
  font-weight: 400;
  src: local('Material Icons'), local('MaterialIcons-Regular'), url(https://fonts.gstatic.com/s/materialicons/v16/2fcrYFNaTjcS6g4U3t-Y5ZjZjT5FdEJ140U2DJYC3mY.woff2) format('woff2');
}
```

**Comment by paulux84** [comment]
I have a commit with this fix...i make a pull request?

**Comment by kevzlou7979** [comment]
Yes make a PR so that we can review. Thanks for the contribution.

**Comment by BenDol** [comment]
@paulux84 that is great news! Put together a PR and we will take a look. Thanks a lot!

**Comment by BenDol** [comment]
@paulux84 can you pull the latest changes and test it? Should fix the issue without using the external url.

**Comment by paulux84** [comment]
#330 pull request done

**Comment by BenDol** [comment]
Can you test the latest version? I have submitted a fix already. Let me know if it works (snapshot is updated).

**Comment by MarcoCucchi** [comment]
@BenDol  it doesn't work on OSX (http://gwtmaterialdesign.github.io/gwt-material-demo/snapshot/#!buttons)

![schermata 2016-05-24 alle 18 05 39](https://cloud.githubusercontent.com/assets/13817916/15511071/508ded3c-21da-11e6-83b9-39dbbe4a11dd.png)

**Comment by BenDol** [comment]
Stay here and wait for my next commit, I have no way of testing it.

**Comment by MarcoCucchi** [comment]
@BenDol ok

**Comment by BenDol** [comment]
@MarcoCucchi are you able to test it on IOS?

**Comment by MarcoCucchi** [comment]
@BenDol nope, I have a Mac

**Comment by BenDol** [comment]
What version of Safari are you using?

**Comment by MarcoCucchi** [comment]
9.0.3

**Comment by MarcoCucchi** [comment]
(11601.4.4)

**Comment by BenDol** [comment]
@MarcoCucchi we will talk in gitter from now on https://gitter.im/GwtMaterialDesign/gwt-material

**Comment by BenDol** [comment]
Resolved for 1.5.1
